modules/view-compliance-status-for-standard.adoc

```diff
@@ -1,7 +1,7 @@
 // Module included in the following assemblies:
 //
 // * operating/manage-compliance.adoc
-:_module-type: PROCEDURE
+:content-type: PROCEDURE
```

Comment:
```suggestion
:_content-type: PROCEDURE
```

Reply:
good grief, I was clearly having brain fog. Thanks for catching these :)